### ISSUE TITLE
Can not run with dotnet

### ISSUE THREAD

**Issue by seanmars** [dotenvx/dotenvx#309]
```sh
dotenvx -V
1.5.0

dotnet --info                                                                                                     
.NET SDK:
 Version:           8.0.302
 Commit:            ef14e02af8
 Workload version:  8.0.300-manifests.011fccd5
 MSBuild version:   17.10.4+10fbfbf2e

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19045
 OS Platform: Windows
 RID:         win-x64
 Base Path:   C:\Program Files\dotnet\sdk\8.0.302\
```

Use the example

```sh
$ dotnet new console -n HelloWorld -o HelloWorld
$ cd HelloWorld
$ echo "HELLO=production" > .env.production
$ echo 'Console.WriteLine($"Hello {Environment.GetEnvironmentVariable("HELLO")}");' > Program.cs
```

then run with

```sh
dotenvx run -f .env -- dotnet run                                                                                  
```

get the error below:

```sh
[dotenvx@1.5.0] missing dotnet file (.\HelloWorld\dotnet)
? add one with [echo "HELLO=World" > dotnet] and re-run [dotenvx run -- ]
[dotenvx@1.5.0] missing run file (.HelloWorld\run)
? add one with [echo "HELLO=World" > run] and re-run [dotenvx run -- ]
[dotenvx@1.5.0] injecting env (1) from .env
missing command after [dotenvx run --]

  get help: [dotenvx help run]
  or try:   [dotenvx run -- npm run dev]
```

**Comment by motdotla** [comment]
if you downgrade to `1.1.0` do you experience the same issue?

```
curl -sfS https://dotenvx.sh/?version=1.1.0
```

**Comment by seanmars** [comment]
> if you downgrade to `1.1.0` do you experience the same issue?
> 
> ```
> curl -sfS https://dotenvx.sh/?version=1.1.0
> ```

Yes, got the same issue

```ps1
[dotenvx@1.1.0] missing dotnet file (.\HelloWorld\dotnet)
? add one with [echo "HELLO=World" > dotnet] and re-run [dotenvx run -- ]
[dotenvx@1.1.0] missing run file (.\HelloWorld\run)
? add one with [echo "HELLO=World" > run] and re-run [dotenvx run -- ]
[dotenvx@1.1.0] injecting env (1) from .env
missing command after [dotenvx run --]

  get help: [dotenvx help run]
  or try:   [dotenvx run -- npm run dev]
```

**Comment by motdotla** [comment]
and are you running this in powershell?

**Comment by seanmars** [comment]
> and are you running this in powershell?

Yes, I'm running this in PowerShell v7

**Comment by seanmars** [comment]
Is there anything new here?

**Comment by motdotla** [comment]
i'll try and take a look this weekend. i haven't made the time to try and replicate it. 

(also, f anyone else is experiencing this, can you please add your +1)

**Comment by motdotla** [comment]
@seanmars we just published to winget. can you try with it:

```
winget install dotenvx
```

(i believe it will remove this error you received, but i think you might have a new issue)

**Comment by motdotla** [comment]
WOW, this took a while to find, but I think it is an encoding issue. on many Windows systems encoding is UTF-16 LE (Little Endian). so `echo "HELLO=World"` is going to encode that way.

i'm updating our docs to have an example that sets encoding utf8, but ideally dotenvx also elegantly handles this or at least warns. i have to think about that.

**Comment by motdotla** [comment]
this is fixed in `1.11.2`, available now and soon on winget (https://github.com/microsoft/winget-pkgs/pull/169930)

**Comment by seanmars** [comment]
@motdotla Try to use the new version(1.11.1), It works correctly and it's awesome.
